avm/res/db-for-postgre-sql/flexible-server/README.md

```diff
@@ -147,7 +147,7 @@ module flexibleServer 'br/public:avm/res/db-for-postgre-sql/flexible-server:<ver
   name: 'flexibleServerDeployment'
   params: {
     // Required parameters
-    name: 'dfpsfsmax001'
+    name: 'dfpfmax001'
```

Comment:
I'd suggest to sneak in a small change in the main.json as well, like a small description update in main.bicep, and then regenerate the json through set-avmmodule utility. 
The reason for that is that as is this PR does not qualify for publishing, so the previous updates won't be published either due the the previous pipeline failure